packages/core-data/src/entities.js

```diff
@@ -182,14 +182,25 @@ export const rootEntitiesConfig = [
 		name: 'globalStyles',
 		kind: 'root',
 		baseURL: '/wp/v2/global-styles',
-		baseURLParams: { context: 'edit' },
+		baseURLParams: {},
```

Comment:
It seems to work ok with `context: edit`, so maybe it's ok to undo these changes.